src/data/staticData/extensionData.ts

```diff
@@ -478,7 +487,16 @@ const extensions: { [key: string]: ExtensionData } = {
       ColonyRole.Funding,
     ],
     enabledExtensionBody: WhitelistExtensionBody,
-    initializationParams: [],
+    initializationParams: [
+      {
+        paramName: 'agreement',
+        validation: yup.string(),
+        defaultValue: '',
+        title: MSG.agreementTitle,
+        description: MSG.agreementDescription,
+        type: ExtensionParamType.Textarea,
+      },
```

Comment:
Good with this, just note, that this is the second parameter passed to the `enableExtension` transaction

Passing it like this will result in an error.